modules/transport-netty4/src/main/java/org/elasticsearch/transport/netty4/Netty4Utils.java

```diff
@@ -55,6 +57,19 @@ public static void setup() {
 
     }
 
+    private static AtomicBoolean isAvailableProcessorsSet = new AtomicBoolean();
+
+    /**
+     * Set the number of available processors that Netty uses for sizing various resources (e.g., thread pools).
```

Comment:
Maybe worth explaining that this does nothing if called more than once. I presume you do that because the plugin is loaded multiple times during testing.

Is it worth logging something if we skip it? If we're outside of testing then it'd be fairly nice to know that it isn't working.